packages/integration-react/api-report.md

```diff
@@ -93,7 +93,7 @@ export interface ScmAuthTokenOptions extends AuthRequestOptions {
       gitlab?: string[];
     };
   };
-  url: string;
+  host: string;
```

Comment:
Good point. I figured it was ok since all impacted usages are updated, but I can imagine that this can also break other plugins.

Reply:
No sorry , I'm talking about the `getCredentials` method. So lets say for GitHub for instance you have two different GitHub Apps configured in config.  One of them is installed in the organization `blob` whilst the other is installed in the organization `boop`.

If I'm just requesting a token for `github.com` then the `getCredentials` method has no idea which app to select as it doesn't have context in the URL as to which org we're trying to reach right? I think thats the motivation behind keeping the URL as you can do `url: `https://github.com/blob/my-repo` to get a token for `my-repo` with the org `blob`.

That's why I think we can't remove it all at least.

Reply:
Yeah, I understand, but how it is implemented right now it completely ignores everything but the host anyways, right?

Reply:
Ah wait sorry, you're right, I'm getting confused here between frontend and backend. It's the `GithubCredentialsProvider` that does that work, and that's backend only.

Ok, maybe there's room for us to do this, but yeah, unsure about the breaking change.

Reply:
I think I can implement this without changing the API. I'll look into it!

Reply:
Yep, should be done!

It doesn't differ too much from the original code and simplifies the PR a lot since the only thing we do is not pass the workspace and repository that go unused anyways.